src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -121,9 +102,29 @@ export const ShoutOutMessage = styled.div<ReactionProps>`
 
   background-color: rgba(255, 255, 255, 1);
   // @debt convert this to scss then use our z-index layer helper here
-  z-index: 12;
+  z-index: ${reactionZIndex};
```

Comment:
I believe emoji reactions and shout-outs explicitly had different layers by choice. This change seems to change/break that.